Resources/doc/index.rst

```diff
@@ -58,20 +58,20 @@ Step 2: Enable the bundle
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Enable the bundle in the kernel:
+ 
+.. code-block:: php
```

Comment:
Oops. Got what you mean.
Sorry, having a bad day :stuck_out_tongue: 
~BadExcuseIKnow